src/class-wpml-elementor-urls.php

```diff
@@ -37,4 +38,21 @@ public function adjust_edit_with_elementor_url( $url, $elementor_document ) {
 
 		return $this->language_converter->convert_admin_url_string( $url, $post_language );
 	}
+
+	/**
+	 * Check if the given URL is the pagination inside the post.
+	 *
+	 * @param bool $is_pagination_url_in_post
+	 * @param string $url URL to check
+	 * @param string $post_name Current post name
+	 *
+	 * @return bool
+	 */
+	public function is_pagination_url( $is_pagination_url_in_post, $url, $post_name ) {
+		return $is_pagination_url_in_post
+		       || (
+			       get_post_meta( get_the_ID(), '_elementor_edit_mode', true )
```

Comment:
@mrudzki91, we have a class constant `WPML_Elementor_Data_Settings::META_KEY_MODE` for this.

And actually, I think you should use this instead:

```php
WPML_Elementor_Data_Settings::is_edited_with_elementor( get_the_ID() )
```

It checks if the post meta is equal to `builder` but I think this is what we actually want in the end.

Reply:
@strategio thanks, I've replaced it. ;)

Reply:
Thanks @mrudzki91 :)